lib/msal-common/src/utils/Constants.ts

```diff
@@ -37,9 +38,20 @@ export const Constants = {
     AUTHORIZATION_PENDING: "authorization_pending",
     NOT_DEFINED: "not_defined",
     EMPTY_STRING: "",
-    FORWARD_SLASH: "/" 
+    FORWARD_SLASH: "/"
 };
 
+export const OIDC_DEFAULT_SCOPES = [
+    Constants.OPENID_SCOPE,
+    Constants.PROFILE_SCOPE,
+    Constants.OFFLINE_ACCESS_SCOPE
+];
+
+export const OIDC_SCOPES = [
+    ...OIDC_DEFAULT_SCOPES,
+    Constants.EMAIL_SCOPE
```

Comment:
Idea for the future, to make this more standards based.  These values can be sourced from the authority config file:

https://login.microsoftonline.com/microsoft.com/v2.0/.well-known/openid-configuration

,"scopes_supported":["openid","profile","email","offline_access"],

Reply:
Will look at adding this in the future, thanks for the suggestion